Resources/Textures/Tiles/Planet/shadowbasalt.rsi/meta.json

```diff
@@ -0,0 +1,66 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "taken from https://github.com/tgstation/tgstation/tree/423404b31e36c4605faacec881b324db8997b0a6 and edited by TheShuEd",
```

Comment:
use permalink not commit

Reply:
I copied the sprites and meta from a neighboring folder of regular basalt. The link taken is the same as from the original.

Reply:
yes and you shouldnt do it since just having the commit doesnt tell you what dmi file its in, just copy permalink for the file then use that